pytorch_lightning/loops/batch/manual.py

```diff
@@ -52,16 +53,16 @@ def advance(self, batch: Any, batch_idx: int) -> None:  # type: ignore[override]
             batch_idx: the index of the current batch
         """
         assert self.trainer is not None
-        ligtning_module = self.trainer.lightning_module
```

Comment:
the previous variable name is much clearer. 
it was always clear that the trainer, lightningmodule etc. objects are always references to memory not literal values. so "ref" in the name is just redundant for basically all Python variable names

Reply:
There's a typo here `ligtning_module` -> `lightning_module`

So since I had to make the fix, I chose to use the same name we have in the `OptimizerLoop`:

https://github.com/PyTorchLightning/pytorch-lightning/blob/58de08d6daf37f9f2f1312a2e21029963ad4e807/pytorch_lightning/loops/optimizer/optimizer_loop.py#L308

But can change both to `lightning_module`.

I thought you prefer `model_ref` as I believe it was first introduced with the accelerator refactor.

Reply:
model ref was a thing before the accelerators even. You are right, I think we should consistently name it the same everywhere. 

It's in the optimizer loop because my PR was a refactor and moving code, not doing any unnecessary renamings.